### DISCUSSION TITLE
Cruiser in Android Auto?

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#371] [Q&A] [answered]
Hi Emux, thanks a lot for the new release 34!! It is amazing how many topics you have solved and new features included!
May I have a new idea? ;-)
In Android Auto there is a list of apps that can be shown on the car display. I don't why these are offered and others not.
Google Maps is my standard app for car navigation, but sometimes - especially on narrow roads - the much more detailled OSM maps would be more useful. And the POIs as well.
Is it possible to offer Cruiser in Android Auto? Or is it an expensive service by Google for the Cruiser App to be accepted in the list?

**Comment by devemux86** [comment, marked as answer]
> thanks a lot for the new release 34!! It is amazing how many topics you have solved and new features included!

@kds112 Thanks for the kind words!

Indeed, this was another big release with new features and tons of improvements.

> Is it possible to offer Cruiser in Android Auto? Or is it an expensive service by Google for the Cruiser App to be accepted in the list?

Android Auto is a different platform with strict requirements. Essentially developers have to write new applications.

Android Auto (and Watch) is in my mind. It's just that right now the mobile app still has a lot of work to do.

**Reply by kds112** [reply]
Okay, I will be patient... 😂 Please put it on the list of long-term ideas...
